src/actions/image_agents.ts

```
@@ -93,13 +94,15 @@ export const imagePreprocessAgent = async (namedInputs: {
   const isMovie = Boolean(beat.moviePrompt || beat?.image?.type === "movie");
-  if (isMovie) {
-    if (beat.soundEffectPrompt) {
+  if (beat.soundEffectPrompt) {
+    if (isMovie) {
       returnValue.soundEffectAgentInfo = MulmoPresentationStyleMethods.getSoundEffectAgentInfo(context.presentationStyle, beat);
       returnValue.soundEffectModel =
         beat.soundEffectParams?.model ?? context.presentationStyle.soundEffectParams?.model ?? returnValue.soundEffectAgentInfo.defaultModel;
       returnValue.soundEffectFile = moviePaths.soundEffectFile;
       returnValue.soundEffectPrompt = beat.soundEffectPrompt;
+    } else {
+      GraphAILogger.warn(`soundEffectPrompt is set, but there is no video. beat: ${index}`)
```

ESLint

[error] Insert `;`

(prettier/prettier)